app/src/test/java/org/mozilla/focus/utils/DownloadUtilsTest.java

```diff
@@ -15,42 +15,56 @@
 @RunWith(RobolectricTestRunner.class)
 public class DownloadUtilsTest {
 
+    private static final int CONTENT_DISPOSITION_TYPES = 2;
+
     private static void assertContentDisposition(String expected, String contentDisposition) {
         Download download = new Download(null, null, contentDisposition, null, 0, null);
         assertEquals(expected, DownloadUtils.guessFileName(download));
     }
 
     @Test
     public void testGuessFileName_contentDisposition() {
-        // Default file name
+        String contentDisposition = "attachment";
+
```

Comment:
\[PMD\] INFO: The String literal "downloadfile\.bin" appears 6 times in this file; the first occurrence is on line 28